policy/modules/kernel/files.if

```diff
@@ -6047,6 +6048,24 @@ interface(`files_manage_var_lib_dirs',`
 	allow $1 var_lib_t:dir manage_dir_perms;
 ')
 
+########################################
```

Comment:
Would be nice to have it in separate commit:
* provide access for /var/run
* provide access for efivars
In this case, it would be easier to apply policies for new platform. For example for u-boot devices we should remove efivar access e.g.

Reply:
Done. Split to several commits